cms/djangoapps/contentstore/rest_api/v2/views/downstreams.py

```diff
@@ -158,16 +162,74 @@ def get_paginated_response(self, data, *args, **kwargs):
         return response
 
 
-@view_auth_classes()
```

Comment:
I'm curious why we don't need this.

Reply:
Sorry, I forgot to add it. Updated https://github.com/openedx/edx-platform/pull/37024/commits/d1ed1d1c7f5397afac3d97f98e6bf911d4968aac